index.js

```diff
@@ -22,6 +22,25 @@ mongoose
   .then(() => {
     // Run your code here, after you have insured that the connection was made
   })
+
+  Recipe.create({ title: "Feijoada", level: "UltraPro Chef", ingredients: ["rice", "chouriço", "morcela", "farinheira", "pork meat", "Masa"], cuisine: "Portugal", dishType: "dinner", image: "https://honest-food.net/wp-content/uploads/2013/12/portuguese-feijoada-320x320.jpg", duration: 120, creator: "Pedro" })
+  .then((recipe) => console.log(`Saved Recipe ${recipe}`))
+
+  .then(() => Recipe.insertMany(data))
+      .then((dataRecipe) =>
+        dataRecipe.forEach((recipes) => {
+          console.log("Create Recipes:", recipes.title);
```

Comment:
You should practice template literals `${}` for your console.logs/return strings, to become more comfortable with it